src/lib/utilities/history-summary-plucker.test.ts

```diff
@@ -0,0 +1,27 @@
+import { getHistorySummary } from './history-summary-plucker';
+
+describe(getHistorySummary, () => {
+  it('should return back unchanged ev', () => {
+    const event = {
+      workflowExecutionStartedEventAttributes: {
+        attempt: 1,
+        workflowRunTimeout: '100s',
+        identity: 'some identity',
+        input: 'true',
+        Parent: undefined,
+        workflowType: {
+          name: 'Rainbow Workflow',
+        },
+      },
+      eventType: 'WorkflowExecutionStarted',
+    };
+
+    expect(getHistorySummary(event)).toStrictEqual({
+      'Close Timeout': '100s',
+      identity: 'some identity',
+      input: 'true',
+      Parent: undefined,
+      Workflow: 'Rainbow Workflow',
+    });
+  });
```

Comment:
When you add types to everything you can check if there are type errors with ts expect error comments

https://devblogs.microsoft.com/typescript/announcing-typescript-3-9-beta/#ts-expect-error-comments